didcomm_resolver/role/requester/routes.py

```diff
@@ -21,7 +21,7 @@
 from aries_cloudagent.storage.error import StorageError, StorageNotFoundError
 from ...resolver import DIDCommResolver
 
-DID_COMM_SPEC_URI = ""  # FIXME: PLS
+DID_COMM_SPEC_URI = "Coming_Soon"  # FIXME: PLS
```

Comment:
The DID Resolution Protocol is here: https://github.com/hyperledger/aries-rfcs/blob/master/features/0124-did-resolution-protocol/README.md
This is the spec we're looking for, right?